docs/standalone/building-on-electrumsv/rest-api.rst

```diff
@@ -19,6 +19,19 @@ Get a list of all available wallets
 :Endpoint: ``http://127.0.0.1:9999/v1/{network}/dapp/wallets``
 :Regtest example: ``http://127.0.0.1:9999/v1/regtest/dapp/wallets``
 
+**Sample Response**
+
+.. code-block::
+
+    {
+        "wallets": [
```

Comment:
Shouldn't this be filtering out the Sqlite internal files and just leaving the wallet file?

Reply:
Yes I suppose it should... will make the change